src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -106,6 +146,11 @@ export const StickyBottomBanner = ({
 	const [SelectedBanner, setSelectedBanner] = useState<React.FC | null>(null);
 	useOnce(() => {
 		const CMP = buildCmpBannerConfig();
+		const puzzlesBanner = buildPuzzlesBannerConfig(
+			CAPI,
```

Comment:
Not sure if this is withing the sccope of this PR or not but we have a convention in DCR of not passing `CAPI`. CAPI is Haauuuge, like Killobytes and Kilobytes of json. So passing it as a prop is a drain on performance and it's also a lot easier to reason about what  thing is doing when you can explicitly see what inputs it has.

Reply:
I think it's a bit too far out of scope for this PR unfortunately- I had a look at it but it would involve changing so much of the code that it seems like it would be better as something for a separate PR.

Reply:
Sure, I don't want to hold up this PR if that refactor is out of scope. It would be great to move away from this pattern at some point though. `CAPI` weighs in at 17Kb...

The pattern we've used to date has been to extract all the required fields out of CAPI in `App.tsx` and then prop drill them down from there.

Looking at things, there are actually only two places where this rule is not followed, `StickyBottomBanner` and `SignInGateSelector` and I think it would be simple to replace the `CAPI` prop with some more explicit ones. I'm happy to pick this up if you like? I quite enjoy tidy up tasks 😃 Once this one is merged I'll pick it up

Reply:
If you've got the time to take a look at it that would be great! Otherwise I'd be happy to stick it in our backlog as a health task.